packages/autorest.python/autorest/multiapi/__init__.py

```diff
@@ -174,10 +173,6 @@ def process(self) -> bool:
             user_specified_default_api=self.user_specified_default_api,
         )
 
-        # In case we are transitioning from a single api generation, clean old folders
```

Comment:
We shall not do such assumption, otherwise it deletes code generated by `multiapi_combiner`